### PR TITLE
Display cancelation message in ManageAttendees

### PR DESCRIPTION
**Pull request by jasozh**
## Summary

Cancelation messages show up now in a modal according to the hi-fis.

Closes:
- [BUG] Add volunteer cancellation messages to ManageAttendees

## Testing

image

### CODE REVIEW

File: backend/src/events/controllers.ts
Comment (akinfelami) [inline backend/src/events/controllers.ts:282]: this is tricky. The manage attendees page does a request of the same type (/event/:eventid) to grab the details for a specific event. But because this flag is set - it returns the attendees with it...which is not needed. The attendees list can grow really large. The screenshot below shows 60.

Screenshot 2024-06-19 at 22 17 47

File: frontend/src/components/organisms/ManageAttendees.tsx
Comment (akinfelami) [inline frontend/src/components/organisms/ManageAttendees.tsx:90]: If this is all we need the eventData for, perhaps we could just make a request that gives us the status of the user from the server?

**Review by akinfelami — CHANGES_REQUESTED** [review]

File: backend/src/events/controllers.ts
Comment (jasozh) [reply backend/src/events/controllers.ts:282]: I agree. I needed the full attendees object for a couple of reasons but I thought about it and realized I could just make new API endpoints that do the same thing. Fixed it on my other PR and I'll merge the changes over here once that's merged.

File: frontend/src/components/organisms/ManageAttendees.tsx
Comment (jasozh) [reply frontend/src/components/organisms/ManageAttendees.tsx:90]: Yep, made a new endpoint (in the other branch) called `GET /events/:eventid/attendees/:userid`

**Pull request merged**
